pkg/common/x509util/keyid.go

```diff
@@ -27,3 +28,21 @@ func GetSubjectKeyID(pubKey interface{}) ([]byte, error) {
 	keyID := sha1.Sum(subjectKeyInfo.SubjectPublicKey.Bytes) //nolint: gosec // usage of SHA1 is according to specification
 	return keyID[:], nil
 }
+
+// SubjectKeyIDToString parse Subject Key ID into string
+func SubjectKeyIDToString(ski []byte) string {
```

Comment:
In https://github.com/spiffe/spire/pull/4216 @rturner3 landed on lowercase hex, no colon